src/operator/nn/mkldnn/mkldnn_pooling.cc

```diff
@@ -150,6 +150,16 @@ mkldnn::pooling_forward::primitive_desc GetPoolingFwd(const PoolingParam &param,
   int pad_l_ = param.pad[1], pad_r_ = param.pad[1];
   int stride_h_ = param.stride[0], stride_w_ = param.stride[1];
 
+  if (param.pooling_convention == pool_enum::kFull) {
```

Comment:
Is it possible to write up a macro/function for the same check of `kFull` in the different place in this file?

Reply:
Done.